smart-contracts/contracts/cl-vault/src/test_tube/any_deposit.rs

```diff
@@ -1,128 +1,176 @@
 #[cfg(test)]
 mod tests {
-    use crate::msg::UserBalanceQueryMsg::{UserAssetsBalance, UserSharesBalance};
-    use crate::msg::{ExecuteMsg, ExtensionQueryMsg};
-    use crate::query::{AssetsBalanceResponse, UserSharesBalanceResponse};
+    use std::str::FromStr;
+
+    use crate::msg::ExecuteMsg;
     use crate::test_tube::initialize::initialize::{
-        fixture_default_less_slippage, ACCOUNTS_INIT_BALANCE, ACCOUNTS_NUM, DENOM_BASE, DENOM_QUOTE,
+        fixture_cw_dex_router, ACCOUNTS_INIT_BALANCE, ACCOUNTS_NUM, DENOM_BASE, DENOM_QUOTE,
+    };
+    use cosmwasm_std::{Addr, Coin, Decimal, Uint128};
+    use osmosis_std::types::cosmos::bank::v1beta1::{
+        QueryAllBalancesRequest, QueryAllBalancesResponse,
     };
-    use cosmwasm_std::Decimal;
-    use cosmwasm_std::{Coin, Fraction};
-    use cw_vault_multi_standard::VaultStandardQueryMsg::VaultExtension;
     use osmosis_std::types::osmosis::poolmanager::v1beta1::SpotPriceRequest;
-    use osmosis_test_tube::{Account, Module, PoolManager, Wasm};
+    use osmosis_test_tube::{
+        Account, Bank, Module, OsmosisTestApp, PoolManager, SigningAccount, Wasm,
+    };
+
+    const MAX_SLIPPAGE: f64 = 0.1;
```

Comment:
this does depend very much on the initial pool size, but yes it does sound very high, I'd expect more 1% max

Reply:
This has been raised to 10% because 1% was a very little amount to generate rewards in our current setup due to underflow. We could be lower it back, and increase the amount of swaps we do on specific tests in order to rebalance the amounts.